tcms/templates/plan/get_cases.html

```diff
@@ -215,27 +186,3 @@
 	</span>
 	<span class="js-nomore-msg" style="display:none">All cases have been loaded.</span>
 </div>
-{% verbatim %}
-	<script id="batch_tag_form_template" type="text/x-handlebars-template">
-		<form id="id_batch_tag_form">
-			<div class="dia_title" style=" margin:30px 20px;">Please type tag name:</div>
-			<div class="dia_content" style=" margin:30px 20px;">
-				<input type="text" id="add_tag_plan" name="tags" style="width:300px; height:25px; border:solid  1px #ccc"/>
-				<div id="id_batch_add_tags_autocomplete"></div>
-			</div>
-			<div style=" margin:30px 20px;">
-				<input type="submit" value="Submit">
-				<input type="button" value="Cancel" class="js-cancel-button" >
-			</div>
-		</form>
-	</script>
-	<script id="batch_tag_summary_template" type="text/x-handlebars-template">
-		<div class="dia_title" style=" margin:10px 20px;">You have successfully updated tags for the following case(s):</div>
-		<div class="dialog_content">
-			{{# each tags }}
-			<div class="dia_content" style=" margin:10px 20px;">{{ this.pk }} &nbsp; {{ this.fields.summary }}</div>
-			{{/ each }}
-		</div>
-		<input class="dia_btn_close sprites js-close-button" type="button" value="Close" style=" margin:10px 20px;"/>
```

Comment:
dia_btn_close is not used anywhere else. Remove it from CSS.

Reply:
Done